format/FlightSql.proto

```diff
@@ -867,6 +867,167 @@ enum SqlSupportsConvert {
+ *                         NULL is returned for data types where a literal suffix is not applicable.),
+ *   create_params: list<utf8 not null>
+ *                        (A list of keywords corresponding to which parameters can be used when creating
+ *                         a column for that specific type.
+ *                         NULL is returned if there are no parameters for the data type definition.),
+ *   nullable: int not null (Shows if the data type accepts a NULL value. The possible values can be seen in the
+ *                           Nullable enum.),
+ *   case_sensitive: bool not null (Shows if a character data type is case-sensitive in collations and comparisons),
+ *   searchable: int not null (Shows how the data type is used in a WHERE clause. The possible values can be seen in the
+ *                             Searchable enum.),
+ *   unsigned_attribute: bool (Shows if the data type is unsigned. NULL is returned if the attribute is
+ *                             not applicable to the data type or the data type is not numeric.),
+ *   fixed_prec_scale: bool not null (Shows if the data type has predefined fixed precision and scale.),
+ *   auto_increment: bool (Shows if the data type is auto incremental. NULL is returned if the attribute
+ *                         is not applicable to the data type or the data type is not numeric.),
+ *   local_type_name: utf8 (Localized version of the data source-dependent name of the data type. NULL
+ *                          is returned if a localized name is not supported by the data source),
+ *   minimum_scale: int (The minimum scale of the data type on the data source.
+ *                       If a data type has a fixed scale, the MINIMUM_SCALE and MAXIMUM_SCALE
+ *                       columns both contain this value. NULL is returned if scale is not applicable.),
+ *   maximum_scale: int (The maximum scale of the data type on the data source.
+ *                       NULL is returned if scale is not applicable.),
+ *   sql_data_type: int not null (The value of the SQL DATA TYPE which has the same values
+ *                                as date_type value. Except for interval and datetime, which
+ *                                uses generic values. More info about those types can be
+ *                                obtained through datetime_subcode. The possible values can be seen
+ *                                in the SqlDataType enum.),
+ *   datetime_subcode: int (Only used when the SQL DATA TYPE is interval or datetime. It contains
+ *                          its sub types. For type different from interval and datetime, this value
+ *                          is NULL. The possible values can be seen in the DateSubcode enum.),
+ *   num_prec_radix: int (If the data type is an approximate numeric type, this column contains
+ *                        the value 2 to indicate that COLUMN_SIZE specifies a number of bits. For
```

Comment:
You mean column_size (lowercase)?

Reply:
Perhaps make column_size a `struct(size: int, radix: int)` instead?

Reply:
The same problem as in this comment: https://github.com/apache/arrow/pull/11982#discussion_r801949555